src/wp-includes/link-template.php

```diff
@@ -2949,14 +2949,20 @@ function get_the_posts_pagination( $args = array() ) {
 			$args['aria_label'] = $args['screen_reader_text'];
 		}
 
+		if ( ! empty( get_queried_object()->labels->name ) ) {
+			$post_type_name = get_queried_object()->labels->name;
+		} else {
+			$post_type_name = 'Posts';
```

Comment:
```suggestion
			$post_type_name = __( 'Posts' );
```

The "Posts" text should be translatable.